src/librustdoc/html/render.rs

```diff
@@ -3625,6 +3626,8 @@ impl<'a> fmt::Display for Sidebar<'a> {
             write!(fmt, "<script defer src=\"{path}sidebar-items.js\"></script>",
                    path = relpath)?;
         }
+        // Closes sidebar-elems div.
+        write!(fmt, "</div>")?;
```

Comment:
Is this `</div>` placed correctly? It is at different indentation level than line 3578. htmldocck on CI currently complains all files are invalid HTML.

```
[01:05:10] stderr:
[01:05:10] ------------------------------------------
[01:05:10] Traceback (most recent call last):
[01:05:10]   File "/checkout/src/etc/htmldocck.py", line 455, in <module>
[01:05:10]     check(sys.argv[1], get_commands(sys.argv[2]))
[01:05:10]   File "/checkout/src/etc/htmldocck.py", line 448, in check
[01:05:10]     check_command(c, cache)
[01:05:10]   File "/checkout/src/etc/htmldocck.py", line 397, in check_command
[01:05:10]     tree = cache.get_tree(c.args[0])
[01:05:10]   File "/checkout/src/etc/htmldocck.py", line 309, in get_tree
[01:05:10]     raise RuntimeError('Cannot parse an HTML file {!r}: {}'.format(path, e))
[01:05:10] RuntimeError: Cannot parse an HTML file 'foo/fn.f.html': pop from empty stack
```

Reply:
Arf, forgot to write the condition... (it's not supposed to be there everytime)